packages/venia-ui/lib/components/RadioGroup/radioGroup.js

```diff
@@ -33,15 +32,15 @@ const RadioGroup = props => {
                     label: classes.radioLabel,
                     root: classes.radioContainer
                 }}
-                id={`${domId}--${value}`}
+                id={`${id}--${value}`}
```

Comment:
Reflected in the snaps, all usages of this component don't pass `id`, so this generates ids like `undefined--value`. That id also gets passed to Informed, but React removes it since it's `undefined`. I propose we either require `id` or remove its usage entirely.